src/dpq2/oids.d

```diff
@@ -175,6 +176,7 @@ OidType detectOidTypeFromNative(T)()
         static if(is(UT == TimeOfDay)){ return Time; } else
         static if(is(UT == SysTime)){ return TimeStampWithZone; } else
         static if(is(UT == dpq2.conv.time.TimeStamp)){ return TimeStamp; } else
+        static if(is(UT == VibeJson)){ return Jsonb; } else
```

Comment:
Should be a `Json`, without `b`? Because we isn't have converter for native json types to PG's `jsonB`